src/objects/explosion.lua

```diff
@@ -6,19 +6,15 @@
 local obj = require 'obj'
 local width = 32
 local height = 32
```

Comment:
Would it be preferable to remove the `width` variable here, as you have done in `nade.lua`?

Reply:
Both of the fields will be removed once the sprite system is merged in.